Classes/Command/CompressImagesCommandController.php

```diff
@@ -53,12 +53,13 @@ public function compressCommand()
         /** @var FileStorage $fileStorage */
         foreach ($this->fileStorageRepository->findAll() as $fileStorage) {
             $excludeFolders = GeneralUtility::trimExplode(',', (string)$settings['exludeFolders'], true);
-            $files = $this->fileRepository->findAllNonCompressedInStorageWithLimit($fileStorage, 100, $excludeFolders);
+            $files = $this->fileRepository->findAllNonCompressedInStorageWithLimit($fileStorage, 1, $excludeFolders);
```

Comment:
Why did you change the limit to 1 file? It should stay at 100 or even better be configurable.

Reply:
argh, sure, this was a testing typo